client/webserver/site/src/js/markets.ts

```diff
@@ -2588,7 +2588,7 @@ class MarketList {
     for (const row of this.markets) {
       if (row.mkt.xc.host !== note.host) continue
       const spot = note.spots[row.mkt.name]
-      if (!spot) return
+      if (!spot) continue
```

Comment:
Can we just delete these two lines? In **app.ts**, we're handling the spot note like
```Typescript
for (const [mktName, spot] of Object.entries(n.spots)) xc.markets[mktName].spot = spot
```
So the `mkt.spot` below is identical to `spot`, and in `setPriceAndChange`, the first line is
```Typescript
if (!mkt.spot) return
```

Reply:
I guess we can, seems to be working same without, applied.